tests/Unit/Middleware/RetryMiddlewareTest.php

```diff
@@ -384,6 +385,45 @@ public function testMaxRetries()
         $middleware($call->reveal(), [])->wait();
     }
 
+    public function testRetriesAreIncludedInTheOptionsArray()
+    {
+        $call = $this->getMockBuilder(Call::class)
+            ->disableOriginalConstructor()
+            ->getMock();
```

Comment:
can we make this prophecy instead?
```suggestion
        $call = $this->prophecy(Call::class)->reveal();
```